content/pl/training/_index.html

```diff
@@ -0,0 +1,108 @@
+      <h2>Kariera <em>Cloud Native</em></h2>
+      <p>Kubernetes stanowi serce całego ruchu <em>cloud native</em>. Korzystając ze szkoleń i certyfikacji oferowanych przez Linux Foundation i naszych partnerów zainwestujesz w swoją karierę, nauczysz się korzystać z Kubernetesa i sprawisz, że Twoje projekty <em>cloud native</em> osiągną sukces.</p>
+    </center>
+  </div>
+</div>
+
+<section>
+  <div class="main-section padded">
+    <center>
+      <h2>Darmowe kursy na edX</h2>
+    </center>
+  <div class="col-container">
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>Wprowadzenie do Kubernetesa<br> &nbsp;</b>
+        </h5>
+        <p>Chcesz nauczyć się Kubernetesa? Oto solidne podstawy do poznania tego potężnego systemu zarządzania aplikacjami w kontenerach.</p>
+        <br>
+        <a href="https://www.edx.org/course/introduction-to-kubernetes" target="_blank" class="button">Przejdź do kursu</a>
+      </center>
+    </div>
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>Wprowadzenie do technologii infrastruktur chmurowych</b>
+        </h5>
+        <p>Poznaj podstawy budowy i zarządzania technologiami chmurowymi bezpośrednio od Linux Foundation – lidera otwartego oprogramowania.</p>
+        <br>
+        <a href="https://www.edx.org/course/introduction-to-cloud-infrastructure-technologies" target="_blank" class="button">Przejdź do kursu</a>
+      </center>
+    </div>
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>Wprowadzenie do Linuksa</b>
+        </h5>
+        <p>Nigdy nie uczyłeś sie o Linuksie? Chcesz odświeżyć swoją wiedzę? Zdobądź praktyczną wiedzę używając interfejsów graficznych oraz linii poleceń w najważniejszych dystrybucjach Linuksa.</p>
+        <br>
+        <a href="https://www.edx.org/course/introduction-to-linux" target="_blank" class="button">Przejdź do kursu</a>
+      </center>
+    </div>
+  </div>
+</section>
+
+<div class="padded lighter-gray-bg">
+  <div class="main-section two-thirds-centered">
+    <center>
+      <h2>Nauka z Linux Foundation</h2>
+      <p>Linux Foundation oferuje szkolenia prowadzone przez instruktora oraz szkolenia samodzielne obejmujące wszystkie aspekty rozwijania i zarządzania aplikacjami na Kubrnetesie.</p>
+      <br><br>
+      <a href="https://training.linuxfoundation.org/training/course-catalog/?_sft_technology=kubernetes" target="_blank" class="button">Zobacz ofertę szkoleń</a>
+    </center>
+  </div>
+</div>
+
+<section>
+  <div class="main-section padded">
+    <center>
+        <h2>Uzyskaj certyfikat Kubernetes</h2>
+    </center>
+    <div class="col-nav">
+      <center>
+        <h5>
+          <b>Certified Kubernetes Application Developer (CKAD)</b>
+        </h5>
+        <p>Egzamin na certyfikowanego dewelopera aplikacji (Certified Kubernetes Application Developer) potwierdza umiejętności projektowania, budowania, konfigurowania i udostępniania "rdzennych" aplikacji dla Kubernetesa.</p>
```

Comment:
nie podoba mi się to "rdzenne", ale nie mam pomysłu na nic lepszego...

Reply:
natywnych?

Reply:
```suggestion
        <p>Egzamin na certyfikowanego dewelopera aplikacji (Certified Kubernetes Application Developer) potwierdza umiejętności projektowania, budowania, konfigurowania i udostępniania aplikacji tworzonych na Kubernetesa.</p>
```

Reply:
"aplikacji rodzimych dla Kubernetesa"?

Reply:
"projektowanie, budowanie ... aplikacji tworzonych na K" - trochę masło maślane z tego wychodzi.